src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Shuffle.txt

```diff
@@ -0,0 +1,5 @@
+>> CountRows(Shuffle([-2, -1, 5, 1, 2]))
+5
+
+>> Sort(Shuffle([-2, -1, 5, 1, 2]), Value)
+[-2,-1,1,2,5]
```

Comment:
Hard to write a test that captures random behavior here. 
Maybe something like

`With({SortedFirstItems: Sort(ForAll(Sequence(2000), First(Shuffle([5,6,7,8,9]).Value))}, First(SortedFirstItems).Value <> Last(SortedFirstItems).Value)`
true

Reply:
if there was a clean way to set the randomizer seed we use, you could also use a `#SETUP: RandomizerTestSetup`
at the top of the file, like OptionSet.txt and make the randomizer seed fixed in a setup handler you write in src\tests\Microsoft.PowerFx.Interpreter.Tests\PowerFxEvaluationTests.cs

Reply:
My concern here is that seeding the random number locks us into a very specific implementation and platform. It'd would for .net (and possibly only a specific build version), but how would a JS test runner use the seed?

Reply:
fwiw, we had a similar challenge for Rand().  Tough to do deterministically especially in a .txt.

Reply:
Yeah, true. I think we have to go with indirectly testing randomness like above then.